lib/matplotlib/axes/_axes.py

```diff
@@ -3282,10 +3282,34 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             x = np.asarray(x, dtype=object)
         if not isinstance(y, np.ndarray):
             y = np.asarray(y, dtype=object)
+
+        def _upcast_errors(err):
```

Comment:
Maybe just `upcast_err`?  Otherwise it sounds like you are going to deal with capital-E Errors...

Reply:
done.